CONTRIBUTING.md

```diff
@@ -0,0 +1,219 @@
+# Contribute to the secrets of sonar-text
+
+## Development environment
```

Comment:
Step 1 and 5 don't bring much value here IMO. It's also not really a step by step guide, more a list of requirements in order to develop new secrets.